source/basic.tex

```diff
@@ -5069,17 +5069,13 @@
 \end{note}
 
 \pnum
-A \defn{release sequence} headed by a release operation \placeholder{A} on an atomic object
-\placeholder{M} is a maximal contiguous sub-sequence of
+A \defn{release sequence} headed
+by a release operation \placeholder{A} on an atomic object \placeholder{M}
+is a maximal contiguous sub-sequence of
 \indextext{side effects}%
-side effects in the
-modification order of \placeholder{M}, where the first operation is \tcode{A}, and
-every subsequent operation
-
-\begin{itemize}
-\item is performed by the same thread that performed \tcode{A}, or
-\item is an atomic read-modify-write operation.
-\end{itemize}
+side effects in the modification order of \placeholder{M},
+where the first operation is \tcode{A}, and
```

Comment:
We have a mixture of `\placeholder{A}` and `\tcode{A}` here. Can you change this from tcode to placeholder?

Reply:
Fixed.